labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -59,7 +59,7 @@ object ListManipulationExercise01 {
    *  - ... etc 
    */
   def concatLists[T](l1: List[T], l2: List[T]): List[T] = {
-    error("fix me")
+    l1.union(l2)
   }
 
```

Comment:
Ah, yeah, makes sense. Test cases weren't all that robust, but I wouldn't be surprised if it does work for lists.